src/python/pants/engine/internals/native.py

```diff
@@ -77,6 +63,8 @@ class Externs:
     def __init__(self, lib: ModuleType):
         self.lib = lib
 
+    # NB: This flag is exposed for testing error handling in CFFI methods. This should never be set
```

Comment:
We should try to modify our tests so we can get rid of this, but thanks for marking it as testing-only in the meantime

Reply:
Most of the exception sink tests are skipped anyway, so hopefully that won't be too difficult.